bmap-writer-test.sh

```diff
@@ -31,6 +31,14 @@ echo "## Write the file with bmap-writer and gzip"
 ./bmap-writer test.img.gz test.img.bmap test.gz.img.out
 cmp test.img.out test.gz.img.out
 
-# echo "## Write the file with bmap-writer and xz"
-# ./bmap-writer test.img.xz test.img.bmap test.xz.img.out
-# cmp test.img.out test.xz.img.out
+echo "## Write the file with bmap-writer and xz"
+./bmap-writer test.img.xz test.img.bmap test.xz.img.out
+cmp test.img.out test.xz.img.out
+
+echo "## Verify the xz decompression test passes"
```

Comment:
the script is running with **/bin/bash -e** it means it will interrupt the execution on the first error  and exit, so it will not reach here.